package.json

```diff
@@ -56,9 +56,9 @@
     "properties": "1.2.1",
     "rbradford-compodoc": "1.1.11",
     "rxjs": "6.5.5",
-    "tsickle": "0.38.1",
-    "tslib": "1.10.0",
-    "zone.js": "0.10.2"
+    "tsickle": "0.39.1",
+    "tslib": "^2.0.0",
```

Comment:
All the dependencies should be exact dependencies. 
There are others, including in other files that should be changed.